config/samples/layered-image-deploy-cm.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  osImageURL: "quay.io/openshift/tdx:ocp415"
```

Comment:
Is this a suitable upstream default value?

Reply:
Not at all ;)